lib/Sema/ConstraintSystem.cpp

```diff
@@ -1434,9 +1434,13 @@ ConstraintSystem::getWrappedPropertyInformation(
 ///   is being accessed; must be null if and only if this is not
 ///   a type member
 static bool
-doesStorageProduceLValue(AbstractStorageDecl *storage, Type baseType,
-                         DeclContext *useDC,
-                         ConstraintLocator *memberLocator = nullptr) {
+doesStorageProduceLValue(
+    AbstractStorageDecl *storage, Type baseType,
+    DeclContext *useDC,
+    llvm::function_ref<bool(Expr *)> isAssignTarget = [](Expr *) {
+      return true;
```

Comment:
Similar to `getUnopenedTypeOfReference` I'm not sure if that's always correct to default to `true`, what if argument is omitted by mistake?

Reply:
I've removed the defaults, thanks